### PR TITLE
Add new icons

### PR DESCRIPTION
**Pull request by germain-gg**
Exports all icons that are listed in the Figma Icons document, https://www.figma.com/file/gkNXqPoiJhEv2wt0EJpew4/Compound-Icons?type=design&node-id=103-5&mode=design&t=xc0vX85VVp77nCMU-11

### CODE REVIEW

**Review by nadonomy — APPROVED** [review]

File: assets/ios/swift/CompoundLightDesignTokens.swift
Comment (pixlwave) [inline assets/ios/swift/CompoundLightDesignTokens.swift:1]: I don't think this file should be generated any more. Is it a left over from a different branch perhaps?

File: assets/ios/swift/CompoundLightHcDesignTokens.swift
Comment (pixlwave) [inline assets/ios/swift/CompoundLightHcDesignTokens.swift:1]: Same for this one too.

File: assets/ios/swift/CompoundDarkHcDesignTokens.swift
Comment (pixlwave) [inline assets/ios/swift/CompoundDarkHcDesignTokens.swift:1]: And this one too.

**Review by pixlwave — APPROVED** [review]
LGTM 👍

**Pull request merged**